src/components/editor/MonacoEditor.tsx

```diff
@@ -0,0 +1,54 @@
+import Editor from '@monaco-editor/react';
```

Comment:
For the most part this file is the same as before - but now we just pass in the stuff it needs instead of having it hardcoded in the file.